pilot/pkg/model/service.go

```diff
@@ -45,6 +46,16 @@ import (
 	"istio.io/istio/pkg/network"
 )
 
+// HostVIPs provides the VIPs in each cluster for a given host.
+type HostVIPs struct {
+	// Name of the service, e.g. "catalog.mystore.com"
+	Hostname host.Name `json:"hostname"`
+
+	// ClusterVIPs specifies the service address of the load balancer
+	// in each of the clusters where the service resides
+	ClusterVIPs cluster.AddressMap `json:"clusterVIPs,omitempty"`
```

Comment:
nit: why not define AddressMap in this pkg

Reply:
It uses `cluster.Hostname`. I chose to move it to that package since it doesn't really have any dependencies on pilot. No real strong motivation.